core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -307,6 +309,13 @@ open class TileInfo {
                     stats.add(unique.stats)
                 }
             }
+            // This entire function should really be refactored, all of these almost similar unique checks are really ugly.
+            // But for that, we would probably first have to generalize _all_ uniques to use cityFilters, and that might take a while
+            for (unique in city.religion.getMatchingUniques("[] from every []")) {
```

Comment:
Just add it to line 307. 
for (unique in city.civInfo.getMatchingUniques("[] from every []")
     + city.civInfo.getMatchingUniques("[] from every []")) ...